src/oss/confluent.sh

```diff
@@ -154,6 +154,20 @@ requirements() {
     if [[ ${status} -ne 0 ]]; then
         FORMAT_CMD="xargs -0"
     fi
+
+    # Validate Java Version
+    local java_version=$(java -version 2>&1 | sed -n ';s/.* version "\(.*\)\.\(.*\)\..*"/\1\2/p;')
+    echo "Using Java Version $java_version"
+    # Warn users if it is not Java7 or Java8, or if we can't figure it out.
+    if [[ $java_version -ne 17 && $java_version -ne 18 ]]; then
+        echo "WARNING: Java version 1.8 is recommended:"
```

Comment:
Inconsistency in formatting of java version: message here says 1.8, message below says 9.